src/app/content/components/Page/highlightManager.ts

```diff
@@ -81,9 +82,24 @@ const createHighlighter = (services: Omit<HighlightManagerServices, 'highlighter
 
   const highlighter: Highlighter = new Highlighter(services.container, {
     formatMessage: (id: string, style?: string) => intl.formatMessage({ id }, { style }),
-    onClick: (highlight) => onClickHighlight({ ...services, highlighter }, highlight),
-    onFocusIn: (highlight) => services.getProp().focus(highlight.id),
-    onFocusOut: () => services.getProp().clearFocus(),
+    onClick: (highlight) => onFocusHighlight({ ...services, highlighter }, highlight),
+    onFocusIn: (highlight) => onFocusHighlight({ ...services, highlighter }, highlight),
+    onFocusOut: () => {
+      defer(() => {
```

Comment:
i don't understand why this would have to be deferred, if you pressed the key combination, sure the movement of the focus is deferred by the hook handling, but focusOut won't be fired until after that is processed right? so by the time we get in here the focus has already been moved?

Reply:
I've dived deeper in this.
We are returning `null` instead of component for cards without annotation that are not focused
then we have a hook that focuses this card if `shouldFocusCard && element.current`
This hook is doing it's job fine but moving focus to this component is causing all cards to be rerendered for some reason 
This is causing `document.activeElement` to be `body`, not the card

This is not happening for other components that are in the dom all the time

---

I've managed to fix it by hiding cards without annotations with css